README.md

```diff
@@ -134,3 +134,7 @@ Please refer to the dedicated [style guide](styleguide.md) and the patterns we d
 ## Contributing
 
 See [how to contribute](CONTRIBUTING.md).
+
+# Tags
+
+* Code for the ION Hackathon demo can be found under the tag `ion-hackaton`
```

Comment:
@ndr-brt I guess this is a spelling mistake. Can you change it to `ion-hackathon`?

Reply:
@juliapampus done thanks